gitarro.rb

```diff
@@ -30,6 +30,3 @@
   break if b.reviewed_pr_test(comm_st, pr)
 end
 STDOUT.flush
-
-# red balls for jenkins
```

Comment:
this is not needed since we exit directly in backend lib

Reply:
https://github.com/openSUSE/gitarro/pull/70/files#diff-8b4041b10595d6631a5e220c1cbeb548R122